sec_certs_page/dashboard/pages/fips.py

```diff
@@ -0,0 +1,26 @@
+"""FIPS 140 collection dashboard page."""
```

Comment:
Not sure where to put this comment, but when I open the FIPS "Add chart" modal I get the "Validation Year" in the x-axis list twice:

<img width="407" height="363" alt="Image" src="https://github.com/user-attachments/assets/10fd1903-2372-4cba-84ca-5fb635a83fa4" />